src/NzbDrone.Core/Qualities/Quality.cs

```diff
@@ -67,12 +67,12 @@ public override bool Equals(object obj)
         public static Quality HDTV1080p => new Quality(9,  "HDTV-1080p");
         public static Quality RAWHD => new Quality(10, "Raw-HD");
         //public static Quality HDTV480p    { get { return new Quality(11, "HDTV-480p"); } }
```

Comment:
Not sure about the group name 'disk 480p', i'd be ok with keeping it DVD.

Remux is more like 'Remux-1080p' and 'Remux-2160p'. If we add them separately then they have their own size limits.
But I don't know which group it should be shoved under. Most people won't even want remuxes. Where previously sizelimits would make sure it didn't, now with separate qualities and thus separate limits (but the same group) it would cause undesired behavior.
It doesn't fall under preferred words, I think that's what we got out of that big debate in the other issue.

Crap, I don't know, I'm feeling like we're adding too many qualities without a means to make it easier for normal users.